build.sbt

```diff
@@ -1,25 +1,33 @@
+val liftVersion = settingKey[String]("Lift Web Framework full version number")
+val liftEdition = settingKey[String]("Lift Edition (such as 2.6 or 3.0)")
```

Comment:
Any reason to have these be `settingKey` instead of just `val`?

Reply:
Building for multiple Scala versions is easy - we can just add `crossScalaVersions` (which I didn't add here because I see it in #26) and it works out of the box. However, I haven't seen so far anything like that for building against different library versions. With `settingKey` we can do just:
```
> set liftVersion:="2.6"
[info] Defining *:liftVersion
[info] The new value will be used by *:libraryDependencies, *:liftEdition
[info] Reapplying settings...
[info] Set current project to lift-formality (in build file:/Users/piotr/git/lift-formality/)

> package
(...)
[info] Packaging /Users/piotr/git/lift-formality/target/scala-2.11/lift-formality_2.6_2.11-1.1.0-SNAPSHOT.jar ...
[info] Done packaging.
```

Reply:
Gotcha. Fair enough!